pkg/stacks/unpack_test.go

```diff
@@ -973,6 +976,54 @@ var (
 	_ StackPackager = &StackPackage{}
 )
 
+func simpleAppFileNoVersion(permissionScope, packageType string) string {
```

Comment:
What do you think about making `simpleAppFile` take a `version` parameter and including the version snippet only when version is not `""`?

Reply:
Hmm yeah. When I was originally considering this I was imagining a boolean for including the version, and I got hung up by the version being in the middle of the file for some reason, but now I think we could just put it at the end instead.